### ISSUE TITLE
Deployment error

### ISSUE THREAD

**Issue by raghavendrakumbhar** [OfficeDev/microsoft-teams-apps-company-communicator#735]
Hi, Deployed the Company Communicator app through powershell. Uploaded the author app through side loading. I am getting login prompt. Upon login got below error.

AADSTS500113: No reply address is registered for the application.

**Comment by gsv022** [comment]
Hi @raghavendrakumbhar ,
There is an ongoing issue with the fsevents library due to which the deployment of App service is failing. Could you please share the logs of the App service deployment? Supporting screenshot of the status will be more helpful.

**Comment by raghavendrakumbhar** [comment]
Sharing the screens, after running powershell script
 - > When we got success message "Finished deploying resources, ARM template deployment succeeded"
 - > When we got failure message "source control deployment failed"

I am also sharing the code snipped which is causing the issue, i think the issues is coming during code sync with Git. attached the code snippet of powershell script which is performing code sync.

Just to know one thing, during first time deployment do we really need the code syn?? , as per script it is required during upgrade only. but not sure whey its getting executed during first time also.

Kindly let know if any other details required. We have lost many days for this deployment and need to fix this earliest to avoid major escalations. 
![screen_3](https://user-images.githubusercontent.com/105361551/170309323-1bc6ec2f-0bf8-48e6-b0cc-31318faa1263.PNG)
![screen_4](https://user-images.githubusercontent.com/105361551/170309386-e7b760d3-c20b-429e-8522-8b7a7b25c9d6.PNG)

![screen_5](https://user-images.githubusercontent.com/105361551/170309468-4882400e-f073-4bac-a66a-804c52d2a046.PNG)

**Comment by gsv022** [comment]
Hi @raghavendrakumbhar ,

The fsevents library is causing the app service deployment to fail. We are working on the fix and will keep you updated. To unblock the deployment please follow the below steps as a quick workaround,

- Go to portal.azure.com. Navigate to resource group where all CC resources are deployed.
- Click on the App service -> Click on Configuration
- Click on **WEBSITE_NODE_DEFAULT_VERSION.**
- Update the default value to **16.13.0** (previous value ->10.15.2)
- Click on Overview and Re-Start the app service.
- Once the app service is restarted. Navigate to **Deployment Center** and click on Sync.
- Wait till the deployment is completed. You can validate this once the status changes to **Success** under logs. 


![image](https://user-images.githubusercontent.com/98143492/170511053-7b2a0cb8-7bc4-477e-b6a6-359012cf6235.png)

![image](https://user-images.githubusercontent.com/98143492/170511111-31010f41-e065-4cd3-a548-075bc61821c3.png)

- For azure function failure, please refer this troubleshooting guide [issue-1] (https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Troubleshooting)

**Comment by gsv022** [comment]
Hi @raghavendrakumbhar ,
Could you please confirm whether the above steps have fixed the issue?

**Comment by raghavendrakumbhar** [comment]
Hi,
I have done the deployment using powershell approach.. The steps mentioned by you above is from manual deployment.. So, is it ok for me to perform the steps you mentioned above even though I did the first time deployment using powershell? Kindly confirm

**Comment by gsv022** [comment]
Hi @raghavendrakumbhar ,
For both manual deployment and powershell deployment, the above suggested steps are applicable. In case of fresh deployment, those steps are not necessary.

**Comment by gsv022** [comment]
Hi @raghavendrakumbhar ,
Thank you for submitting this issue, as we work to ensure consistent responses with items logged in our issues list, we are working to address these items that did not get responses. Please let us know if you are still facing this issue and require assistance and we will be happy to discuss. If no further action is needed at moment and no response is received, we will close out the issue in 3 business days. You may also always open a new issue or respond here if something does come back up.
